### DISCUSSION TITLE
SplitMNIST Benchmark

### DISCUSSION THREAD

**Discussion by ZaidRZ** [ContinualAI/avalanche#1247] [Q&A] [answered]
I have new problem it was not found, that when i call SplitMNIST benchmark  by using 
_**from avalanche.benchmarks.classic import SplitMNIST**_

get this ImportError: cannot import name 'Kinetics400' from 'torchvision.datasets' (/usr/local/lib/python3.8/dist-packages/torchvision/datasets/__init__.py)

**Comment by vlomonaco** [comment, marked as answer]
Dear @ZaidRZ, this has been fixed here: https://github.com/ContinualAI/avalanche/issues/1233
The 0.3.0 release will be uploaded soon on pypi!

**Reply by ZaidRZ** [reply]
thank you so much
